stl/src/wlocale.cpp

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-// class locale wide member functions

```

Comment:
I'm not sure if this is SHOUTY or useless

Reply:
Urgh, not sure how that happened.